src/Data/Nat/DivMod/Core.agda

```diff
@@ -128,28 +128,11 @@ private
   div-cong₃ : ∀ {c n a₁ a₂ b} → a₁ ≡ a₂ → divₕ c n a₁ b ≡ divₕ c n a₂ b
   div-cong₃ refl = refl
 
-  divₕ-restart : ∀ {acc} d n j → j < n → divₕ acc d n j ≡ divₕ (suc acc) d (n ∸ suc j) d
```

Comment:
Out of interest, why have you rearranged the proofs in this file? Are there actually any changes? If not, could you revert them back to how they were. No sense in messing up the Git history unnecessarily!

Reply:
The `divₕ-restart` and `divₕ-finish` proofs are being used in the `m<n⇒m/n≡0` and `m/n≡1+[m∸n]/n` proofs in the other file, but they were in the private section before.

Reply:
Ah thanks, missed that!